README.md

```diff
@@ -295,6 +295,37 @@ custom:
 
 > Note: CloudFormation stack outputs and logical IDs will be changed from the defaults to api name prefixed. This allows you to differentiate the APIs on your stack if you want to work with multiple APIs.
 
+### Direct Lambda Resolvers
+
+Amazon supports [direct lambda resolvers](https://docs.aws.amazon.com/appsync/latest/devguide/direct-lambda-reference.html)
+
+With a direct lambda resolver, no VTL mapping template is required for either request or response. This can be an option if you would like to avoid usage of the Apache VTL langauge or require a complex resolver. Direct lambda resolvers can be enabled for all mappingTemplates with;
+
+```yml
+custom:
+  appsync:
+    defaultMappingTemplates:
+      request: false
+      response: false
+```
```

Comment:
@MarcusJones I would avoid documenting this. As it would work, it can be confusing because `defaultMappingTemplates` applies to ALL resolver types (DynamoDB, HTTP, etc), not only Lambda.

I like the idea to document direct lambda as there currently is only a note as a comment in the Example.
For now, I would just explain to set `false` to each individual Lambda function that requires Direct Lambda support, as you did below.